src/materialized/src/bin/materialized/main.rs

```diff
@@ -147,6 +147,9 @@ struct Args {
     /// Default frequency with which to advance timestamps
     #[structopt(long, env = "MZ_TIMESTAMP_FREQUENCY", hidden = true, parse(try_from_str =repr::util::parse_duration), value_name = "DURATION", default_value = "1s")]
     timestamp_frequency: Duration,
+    /// Default frequency with which to scrape prometheus metrics
+    #[structopt(long, env = "MZ_METRICS_SCRAPING_FREQUENCY", hidden = true, parse(try_from_str = parse_optional_duration), value_name = "DURATION", default_value = "1s")]
```

Comment:
I feel like maybe we'll want to bump the duration here up a bit - I don't think it's very productive to have it ingest all the metrics every second (using up memory uselessly for most people). How's 10s sound?

Reply:
I intended to keep splitting the configuration and changing the default separate, so will follow-up with another PR!